scripts/utility/Convert_Main2Beta_coreNames.sh

```diff
@@ -0,0 +1,128 @@
+#!/usr/bin/env bash
+# See the NOTICE file distributed with this work for additional information
+# regarding copyright ownership.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+## A short script to programmatically transform main side CoreDBs to their appropriate CoreDB name for moving to Beta
+## It takes into account the appropriate assembly accession, annotation source and prefix
+
+MAIN_CORES_IN=$1
+SCHEMA_FOR_BETA=$2
+CMD_HOST=$3
+OPTIONAL_CORE_PREFIX=$4
+if [[ -z $MAIN_CORES_IN ]] || [[ -z $SCHEMA_FOR_BETA ]] ||  [[ -z $CMD_HOST ]]; then
+echo "Usage Convert_Main2Beta_coreNames.sh <list of Ensembl Main cores DBs> <Beta schema required> <Staging host> <*Optional_beta_core_prefix>"
+exit 1
+fi
+OUTPUT_TRANSFORM="Beta_Cores.out.tsv"
+if [[ -f $OUTPUT_TRANSFORM ]]; then
+    rm ./$OUTPUT_TRANSFORM
+fi
+echo -e -n "#EnsemblMain_CoreDB\tEnsembl BetaCoreDB [schema version:${SCHEMA_FOR_BETA}]\n" > $OUTPUT_TRANSFORM
+
+if [[ -f main_to_beta_convert.log ]]; then
+    rm ./main_to_beta_convert.log
+fi
+if [[ -n $OPTIONAL_CORE_PREFIX ]]; then
+    OPTIONAL_CORE_PREFIX="${4}_"
+    echo "Prepending optional core prefix '${OPTIONAL_CORE_PREFIX}'" | tee main_to_beta_convert.log
+    sleep 2
+fi
+
+function convert_main_to_beta(){
+
+        local INPUT_CORES=$1
+        local SUFFIX=$2
+        local STAGING_HOST=$3
+
+    while read CORE
+        do
```

Comment:
This is a preference I have just to "gain" one indentation block:
```suggestion
    while read CORE; do
```